### ISSUE TITLE
Prevent duplicates doesn't work

### ISSUE THREAD

**Issue by angel1st** [Foxandxss/angular-toastr#132]
The `preventDuplicates` config option doesn't work - even in your own example - http://foxandxss.github.io/angular-toastr/. When it is set to `true` no message is shown.

In original toastr it works.

Best Regards,
Angel

**Comment by Foxandxss** [comment]
I see it working. Could you please create a plunker showing what you mean?

If I set true on preventDuplicates, it won't open the same toast again (until you actually create a different toast).

**Comment by angel1st** [comment]
This is not the original behavior inherited from **toastr** functionality - this is the except from https://github.com/CodeSeven/toastr `preventDuplicates` - ***Rather than having identical toasts stack, set the preventDuplicates property to true. Duplicates are matched to the previous toast based on their message content.***, meaning - that only one message with identical **content** is allowed to be shown at time. If it hides, you can reopen it again. The point on this property is to prevent more that one identical message to be displayed at the same time, no to prevent to be displayed for ever.

Refer to http://codeseven.github.io/toastr/demo.html for comparison.

Best Regards,
Angel

**Comment by Foxandxss** [comment]
I have that functionality in `preventOpenDuplicates`.

The original idea was to have a 1:1 port with `toastr` but that is impossible.

They have X features when I decided to make a port, I can offer those X features.

Then there is one original feature (now also in my lib) that has an issue or perhaps it is not behaving correctly. Some guy reports it on my lib, and if I decide to fix it, I am actually diverging from the original implementation.

So after 2 years, it is hard to keep up with the original implementation. I don't recall if originally `preventDuplicates` on the original lib behave like mine and that was changed afterwards or not. I had to create a `preventOpenDuplicates` to achieve that.

There are also issues that only applies to Angular, so I had to diverge on that too.

TL;DR You can't think about this library as a 1:1 port anymore.

I want to create a 2.0 and I could do a update on what they have, but I am not really sure if it worth it.

Cheers.

**Comment by angel1st** [comment]
Thanks man, that's all I needed,
Cheers
